beacon-chain/rpc/validator/status.go

```diff
@@ -30,19 +30,22 @@ var errPubkeyDoesNotExist = errors.New("pubkey does not exist")
 //  UNKNOWN_STATUS - validator does not have a known status in the network.
 func (vs *Server) ValidatorStatus(
 	ctx context.Context,
-	req *ethpb.ValidatorStatusRequest) (*ethpb.ValidatorStatusResponse, error) {
+	req *ethpb.ValidatorStatusRequest,
+) (*ethpb.ValidatorStatusResponse, error) {
 	headState, err := vs.HeadFetcher.HeadState(ctx)
 	if err != nil {
 		return nil, status.Error(codes.Internal, "Could not get head state")
 	}
-	return vs.validatorStatus(ctx, req.PublicKey, headState), nil
+	vStatus, _ := vs.validatorStatus(ctx, headState, req.PublicKey)
```

Comment:
Unhandled error

Reply:
That is the return indice, the function doesn't return an error.